acquisitions/src/main/resources/thunderjet/mod-orders/features/routing-list-print-template.feature

```diff
@@ -0,0 +1,161 @@
+
+    Scenario: Create two users with different addressType are related to routingLists
+      * configure headers = headersAdmin
+      * def user1 = read('classpath:samples/mod-users/' + userId1 + '.json')
+      * set user1.personal.addresses[0].addressTypeId = officeAddressTypeId
+      * set user1.personal.addresses[0].addressLine1 = officeAddressLine1V1
+      * set user1.personal.addresses[1].addressTypeId = homeAddressTypeId
+      * set user1.personal.addresses[1].addressLine1 = homeAddressLine1V1
+      * def user2 = read('classpath:samples/mod-users/' + userId2 + '.json')
+      * set user2.personal.addresses[0].addressTypeId = officeAddressTypeId
+      * set user2.personal.addresses[0].addressLine1 = officeAddressLine1V2
+      * set user2.personal.addresses[1].addressTypeId = homeAddressTypeId
+      * set user2.personal.addresses[1].addressLine1 = homeAddressLine1V2
+
+      Given path 'users'
+      And request user1
+      When method POST
+      Then status 201
+      And match response.personal.addresses[*].addressTypeId contains officeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains officeAddressLine1V1
+      And match response.personal.addresses[*].addressTypeId contains homeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains homeAddressLine1V1
+
+      Given path 'users'
+      And request user2
+      When method POST
+      Then status 201
+      And match response.personal.addresses[*].addressTypeId contains officeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains officeAddressLine1V2
+      And match response.personal.addresses[*].addressTypeId contains homeAddressTypeId
+      And match response.personal.addresses[*].addressLine1 contains homeAddressLine1V2
+
+    Scenario: Create composite order to use in routing list
+      Given path 'orders/composite-orders'
+      And request
+        """
+        {
+          id: '#(orderId)',
+          vendor: '#(globalVendorId)',
+          orderType: 'One-Time'
+        }
+        """
+      When method POST
+      Then status 201
+
+    Scenario: Create order line for composite order that will be used in routingList
+      Given path 'orders/order-lines'
+
+      * def orderLine = read('classpath:samples/mod-orders/orderLines/minimal-order-line.json')
+      * set orderLine.id = poLineId
+      * set orderLine.purchaseOrderId = orderId
+
+      And request orderLine
+      When method POST
+      Then status 201
+      * def response = $
+      And match response.paymentStatus == "Pending"
+
+    Scenario: Create Routing Lists with two user 'user1', 'user2'
+      * def routingList = read('classpath:samples/mod-orders/routingLists/' + routingListId + '.json')
+      * set routingList.id = routingListId
+      * set routingList.poLineId = poLineId
+      * configure headers = headersAdmin
+
+      Given path 'orders-storage/routing-lists'
```

Comment:
url will be changed after https://github.com/folio-org/mod-orders/pull/879 pr merged